### PR TITLE
add the ability to handle, parse and forward issue comments.

### PR DESCRIPTION
**Pull request by jeremyplichtafc**
This will allow us to auto retrigger jenkins branch tests with phrases like "retest this please" on the PR.

`go fmt` was also run, which is why there are a lot of other non related formatting changes

### CODE REVIEW

File: pkg/providers/github.go [hunk 1]
```diff
@@ -68,28 +69,36 @@ func (p *GithubProvider) Validate(hook Hook) bool {
...
 	switch eventType {
 	case GithubPushEvent:
+		if err := json.Unmarshal(hook.Payload, &pushPayloadData); err != nil {
+			log.Printf("Github payload unmarshaling failed for Push event: %v", err)
+			return ""
+		}
 		return pushPayloadData.Sender.Login
 	case GithubPullRequestEvent:
+		if err := json.Unmarshal(hook.Payload, &pullRequestPayloadData); err != nil {
+			return ""
```
Comment (hamza3202) [inline pkg/providers/github.go]: @jeremyplichtafc perhaps an error log here as well as to why pull marshaling of pull request even failed?

**Comment by stakater-user** [comment]
@jeremyplichtafc Image is available for testing. ``docker pull stakater/gitwebhookproxy:SNAPSHOT-PR-69-1``

File: pkg/providers/github.go (hunk 1 above)
Comment (jeremyplichtafc) [reply pkg/providers/github.go]: good catch. got missed when I was reorganizing this section. fixed

**Comment by stakater-user** [comment]
@jeremyplichtafc Image is available for testing. ``docker pull stakater/gitwebhookproxy:SNAPSHOT-PR-69-3``

**Comment by hamza3202** [comment]
Thanks for contributing :-)

**Pull request merged**